### DISCUSSION TITLE
NocoDB SDK generation on api change [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2020] [Discord Threads]
> 🙋‍♂️ **Po**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Api.ts generated from here   
####    
#### https://github.com/nocodb/nocodb/blob/develop/packages/nocodb-sdk/package.json#L40  
#  
> 🙋‍♂️ **Po**  
#### Thank you so much!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968861906951602216)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
